store/retrylayer/retrylayer.go

```diff
@@ -6468,6 +6468,27 @@ func (s *RetryLayerPostStore) GetDirectPostParentsForExportAfter(limit int, afte
 
 }
 
+func (s *RetryLayerPostStore) GetEditHistoryForPost(postId string) ([]*model.Post, error) {
```

Comment:
This Pattern looks like it matches the rest of the the file.  Not for this PR, but seems we could refactor and reduce quite a bit of the duplicated code sometime in the future :)

Reply:
I agree that there are many duplicated code in this file.